### PR TITLE
Issue1113 by pjohans: show sort method in searchresult

### PR DESCRIPTION
**Pull request by pjohans**
this one works with:
https://github.com/ding2/ting-client/pull/8

### CODE REVIEW

File: plugins/content_types/search_result.inc [hunk 1]
```diff
@@ -83,10 +83,22 @@ function ting_search_search_result_content_type_render($subtype, $conf, $panel_a
...
     $message = '';
+
+    $frequency_rank = array(
+      'rank_main_title' => t('Title'),
+      'rank_creator' => t('Creator'),
+      'rank_subject' => t('Subject')
+    );
+    if (isset($search_result) && in_array($search_result->sortUsed, array_keys($frequency_rank))) {
+      $msg = t('Records are sorted by %sortUsed . Select another sort if it is not suited', array('%sortUsed' => $frequency_rank[$search_result->sortUsed]));
+      $message .= '<div class="messages status">' . $msg . '</div>';
```
Comment (kasperg) [inline plugins/content_types/search_result.inc]: I do not see a reason to wrap the message in two `<div>` tags each with the `message` class.

If the `status` class is the reason for doing this then I suggest that we add that class to the div created on line 101 when needed e.g. by introducing a `$message_classes` variable.

```php
$message = t('Records are sorted by %sortUsed . Select another sort if it is not suited', array('%sortUsed' => $frequency_rank[$search_result->sortUsed]));
$message_classes[] = 'status:
```

**Comment by kasperg** [comment]
@pjohans: Review complete.

File: plugins/content_types/search_result.inc (hunk 1 above)
Comment (pjohans) [inline plugins/content_types/search_result.inc]: But i would like the two messages to be shown with different styling eg. :
this one as a warning (yellow) :
"Your search gave more than 100 results. Try to search more specific or use the facets to filter the result."

and this one as a status (green):
"Records are sorted by Title . Select another sort if it is not suited"

Comment (kasperg) [inline plugins/content_types/search_result.inc]: You are right. I did not spot the fact that is is separate strings being concatenated.

**Comment by kasperg** [comment]
This part of the review is approved and ready to be merged.

**Pull request merged**